decidim-consultations/config/locales/en.yml

```
@@ -221,6 +221,9 @@ en:
           success: Response successfully updated.
       titles:
         consultations: Consultations
+        new_consultations: New consultations
```

Let's use singular here:
```suggestion
        new_consultation: New consultation
```